prototype/osl-readme.md

```diff
@@ -98,10 +53,16 @@ In general, any time you pull changes it's a good idea to run both of the above
 
 ## Run the example `water-extent-map.ipynb`
 
-Once you've completed the above one-time setup and restarted your OSL server, you
-can run through the example `water-extent-map.ipynb` notebook, which will be located
-at `${home}/asf-tools/prototype/`
+Once you've completed the above one-time setup, you can run through the example notebook,
+which will be located in the `prototype/` directory:
+* `water-extent-map.ipynb` -- Run the water map code directly. This allows you to
+  edit the code and see the changes reflected.
+* `water-extent-map-sdk.ipynb` -- Request water map products from HyP3 using the
+  `asf_search` python package and the HyP3 SDK.
+* `water-extent-map-vertex.ipynb` -- Request water map products from HyP3 using
+  ASF Search Vertex and the HyP3 SDK.
```

Comment:
Need an update here to reflect these being combined into `water-extent-map-on-demand.ipynb`

Reply:
Oh right! Done.